# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/cache/FileUtils.h
# Commit: fbfdd8ee1b42
# Full Hash: fbfdd8ee1b42fb2d863b1d669a6047087d7e0595
# Author: Tom Tung <shes050117@gmail.com>
# Date: 2019-09-28 09:46:05
# Regressor Bug: 1581067
# File Overlap Count: 1
# Description:
#   Bug 1581067 - P1 - Remove the emptry body directory when removing files inside; r=asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D46586
# ==============================================================================

diff -r 2416d17d33c1 -r fbfdd8ee1b42 dom/cache/FileUtils.h
--- a/dom/cache/FileUtils.h	Fri Sep 27 15:26:14 2019 +0000
+++ b/dom/cache/FileUtils.h	Fri Sep 27 16:54:22 2019 +0000
@@ -64,9 +64,11 @@
 
 bool MarkerFileExists(const QuotaInfo& aQuotaInfo);
 
-nsresult RemoveNsIFileRecursively(const QuotaInfo& aQuotaInfo, nsIFile* aFile);
+nsresult RemoveNsIFileRecursively(const QuotaInfo& aQuotaInfo, nsIFile* aFile,
+                                  const bool aTrackQuota = true);
 
-nsresult RemoveNsIFile(const QuotaInfo& aQuotaInfo, nsIFile* aFile);
+nsresult RemoveNsIFile(const QuotaInfo& aQuotaInfo, nsIFile* aFile,
+                       const bool aTrackQuota = true);
 
 void DecreaseUsageForQuotaInfo(const QuotaInfo& aQuotaInfo,
                                const int64_t& aUpdatingSize);